server/tests/controllers/test_contributors_controller.py

```diff
@@ -0,0 +1,75 @@
+import json
+
+import pytest
+
+
+@pytest.mark.parametrize("cached", [False, True], ids=["no cache", "with cache"])
+async def test_get_contributors_as_admin(client, cached, headers, app, cache):
+    await _test_get_contributors(client, cached, headers, app, cache)
+
+
+@pytest.mark.parametrize("cached", [False, True], ids=["no cache", "with cache"])
+async def test_get_contributors_as_non_admin(client, cached, headers, app, cache, eiso):
```

Comment:
@vmarkovtsev At the beginning it wasn't very clear to me that by just passing `eiso` would have changed the default user.

Reply:
Yeah, it is the only way to avoid calling Auth0 in the unit tests.

Reply:
yup, I mean that probably it could be more explicit than just passing a variable to a function that is then never used, at least to me is counter-intuitive 😄.

Reply:
I am unused variable 😂